packages/v4/src/content/accessibility/alert-group/alert-group.md

```diff
@@ -0,0 +1,56 @@
+---
+id: Alert group
+section: components
+---
+
+An **alert group** positions one or multiple alerts in a layer over the main content of a page. Use an alert group as a pre-existing container for dynamic alerts or when you need to position multiple alerts on the same page, mainly [toast alerts](https://www.patternfly.org/v4/components/alert/design-guidelines/#elements). 
+
+**Static inline alert groups** don’t need additional accessibility attributes. If it appears inline on the page, it will be read along with the surrounding content. 
+
+**Dynamic and toast alerts** are injected into the DOM after page load and require additional measures to ensure that these content changes are announced to screen reader users. Discoverability is also different in this case because the alert content is often not near the primary markup/content for the page. The AlertGroup/Alerts are often rendered at a more global level within the DOM, like toward the end of the body tag so that the content can persist across page changes. 
```

Comment:
```suggestion
**Dynamic and toast alerts** are injected into the DOM after page load and require additional measures to ensure that these content changes are announced to screen reader users. This is the use case that requires/warrants turning an alert group into a live region. Discoverability is also different in this case because the alert content is often not near the primary markup/content for the page. The AlertGroup/Alerts are often rendered at a more global level within the DOM, like toward the end of the body tag so that the content can persist across page changes. 
```